### PR TITLE
Add guide to edit a secret

### PR DESCRIPTION
**Pull request by lvarin**
@amahsanferoz please review this change. I am not sure if it will be clear for the customers.

### CODE REVIEW

**Review by tfrisk — APPROVED** [review]
It's a starting point on difficult topic.

File: docs/cloud/rahti/tutorials/configuration.md [hunk 1]
```diff
@@ -81,4 +81,34 @@ data:
...
+
+* Encode the new file and replace the previous value in the JSON file:
+
+```sh
+B64=$(base64 $KEY_NAME.file -w0)
+jq " .data.$KEY_NAME = \"$B64\" " secret.json
+oc replace -f secret.json
+```
+
+As you can the process can be a bit ofuscated.
```
Comment (junousi) [inline docs/cloud/rahti/tutorials/configuration.md:113]: +see
+obfuscated

Comment (lvarin) [reply docs/cloud/rahti/tutorials/configuration.md]: Fixed, thank you

Comment (junousi) [reply docs/cloud/rahti/tutorials/configuration.md]: The "+see" was probably a bit ambiguous so

s/As you can/As you can see/

Comment (lvarin) [reply docs/cloud/rahti/tutorials/configuration.md]: :sweat: 
fixed

**Review by junousi — APPROVED** [review]
LGTM

**Pull request merged**